worker-test/src/test/java/com/hpe/caf/worker/workertest/GetWorkerNameIT.java

```diff
@@ -44,7 +46,12 @@ public void getWorkerNameInPoisonMessageTest() throws IOException, TimeoutExcept
 
             final Channel channel = connection.createChannel();
 
-            channel.queueDeclare("testworker-out", true, false, false, Collections.emptyMap());
+            Map<String, Object> args = new HashMap<>();
+            final String rabbitQueueTypeName = !Strings.isNullOrEmpty(RABBIT_PROP_QUEUE_TYPE_NAME) ?
+                    RABBIT_PROP_QUEUE_TYPE_NAME : QueueCreator.RABBIT_PROP_QUEUE_TYPE_QUORUM;
```

Comment:
!Strings.isNullOrEmpty(RABBIT_PROP_QUEUE_TYPE_NAME) ?
                    RABBIT_PROP_QUEUE_TYPE_NAME : QueueCreator.RABBIT_PROP_QUEUE_TYPE_QUORUM;
This can be part of the static variable definition on line 40.